pkgs/development/tools/dyff/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, fetchFromGitHub}:
+
+let
+  version = "1.4.3";
+in
+
+buildGoModule rec {
+  inherit version;
```

Comment:
You can directly use `version = "1.4.3"` here, and remove the let ... in as `rec` mean (in `buildGoModule rec`) mean that child can directly access property of this attrset (so `rev = "v${version}"` will work)

Reply:
Thank you for your feedback. I have made the suggested change.